elide-async/src/main/java/com/yahoo/elide/async/service/AsyncQueryCancelThread.java

```diff
@@ -8,6 +8,7 @@
 import static com.yahoo.elide.core.EntityDictionary.NO_VERSION;
 
 import com.yahoo.elide.Elide;
+import com.yahoo.elide.async.models.AsyncAPI;
```

Comment:
Is this class will be common for both AsyncQuery and TableExport - we need to name all the common classes with the same convention - AsyncAPI or AsyncBase or something along those lines.